pkg/lib/models/models.go

```diff
@@ -34,33 +30,17 @@ import (
 //			- variables.index
 //			- variables.data-00000-of-00001 (there are a variable number of these files)
 func IsValidS3Directory(path string) bool {
```

Comment:
What do you think about moving this function into the `userconfig` package, and later we can move it out if other packages need it? It could go either way, up to you